### PR TITLE
Gasping in shock, but for non-humans!

### PR DESCRIPTION
**Pull request by dis-integrates-the-integration-tests**
## About The Pull Request

This PR _should_ make *gaspshock available for non-human species.

## How This Contributes To The Skyrat Roleplay Experience

*gaspshock used to be limited to humans only, which.. did not make much sense.
Now everyone can gasp in shock!

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>

  https://cdn.discordapp.com/attachments/640760727489085450/1182094163748061284/Is6ZPUT.mp4?ex=658371fa&is=6570fcfa&hm=d3b87cbbafb5b260668c10602620df11d166029acc94674dc59f87ea69ece903&
</details>

## Changelog

:cl: KannaLisvern
fix: *gaspshock being only limited to humans
/:cl:

### CODE REVIEW

File: code/modules/mob/living/emote.dm [hunk 1]
```diff
@@ -205,11 +205,12 @@
...
-		if(human_user.physique == FEMALE)
-			return pick('sound/voice/human/gasp_female1.ogg', 'sound/voice/human/gasp_female2.ogg', 'sound/voice/human/gasp_female3.ogg')
-		else
-			return pick('sound/voice/human/gasp_male1.ogg', 'sound/voice/human/gasp_male2.ogg')
+	if(HAS_MIND_TRAIT(human_user, TRAIT_MIMING))
+		return
+	if(human_user.physique == FEMALE)
+		return pick('sound/voice/human/gasp_female1.ogg', 'sound/voice/human/gasp_female2.ogg', 'sound/voice/human/gasp_female3.ogg')
+	else
+		return pick('sound/voice/human/gasp_male1.ogg', 'sound/voice/human/gasp_male2.ogg')
```
Comment (vinylspiders) [inline code/modules/mob/living/emote.dm]:
```suggestion
	if(ishuman(human_user) && !HAS_MIND_TRAIT(human_user, TRAIT_MIMING)) // SKYRAT EDIT CHANGE - ORIGINAL: if(human_user.dna.species.id == SPECIES_HUMAN && !HAS_MIND_TRAIT(human_user, TRAIT_MIMING))
		if(human_user.physique == FEMALE)
			return pick('sound/voice/human/gasp_female1.ogg', 'sound/voice/human/gasp_female2.ogg', 'sound/voice/human/gasp_female3.ogg')
		else
			return pick('sound/voice/human/gasp_male1.ogg', 'sound/voice/human/gasp_male2.ogg')
```
These are TG files, when editing those every single diff _must_ be labeled as a Skyrat edit. https://github.com/Skyrat-SS13/Skyrat-tg/blob/master/modular_skyrat/readme.md

**Review by vinylspiders — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/emote.dm (hunk 1 above)
Comment (dis-integrates-the-integration-tests) [reply code/modules/mob/living/emote.dm]: My bads, fixing rn

File: code/modules/mob/living/emote.dm [hunk 2]
```diff
@@ -205,7 +205,8 @@
 	if(!ishuman(user))
 		return
 	var/mob/living/carbon/human/human_user = user
-	if(human_user.dna.species.id == SPECIES_HUMAN && !HAS_MIND_TRAIT(human_user, TRAIT_MIMING))
+	if(ishuman(human_user) && HAS_MIND_TRAIT(human_user, TRAIT_MIMING)) // SKYRAT EDIT CHANGE - ORIGINAL: if(human_user.dna.species.id == SPECIES_HUMAN && !HAS_MIND_TRAIT(human_user, TRAIT_MIMING))
```
Comment (vinylspiders) [inline code/modules/mob/living/emote.dm]:
```suggestion
	if(ishuman(human_user) && !HAS_MIND_TRAIT(human_user, TRAIT_MIMING)) // SKYRAT EDIT CHANGE - ORIGINAL: if(human_user.dna.species.id == SPECIES_HUMAN && !HAS_MIND_TRAIT(human_user, TRAIT_MIMING))
```
One thing here

**Review by vinylspiders — CHANGES_REQUESTED** [review]

**Review by vinylspiders — APPROVED** [review]

File: code/modules/mob/living/emote.dm [hunk 3]
```diff
@@ -205,7 +205,8 @@
 	if(!ishuman(user))
 		return
 	var/mob/living/carbon/human/human_user = user
-	if(human_user.dna.species.id == SPECIES_HUMAN && !HAS_MIND_TRAIT(human_user, TRAIT_MIMING))
+	if(ishuman(human_user) && !HAS_MIND_TRAIT(human_user, TRAIT_MIMING)) // SKYRAT EDIT CHANGE - ORIGINAL: if(human_user.dna.species.id == SPECIES_HUMAN && !HAS_MIND_TRAIT(human_user, TRAIT_MIMING))
+		return
```
Comment (vinylspiders) [inline code/modules/mob/living/emote.dm]:
```suggestion
```
Err two things...

File: code/modules/mob/living/emote.dm [hunk 4]
```diff
@@ -205,7 +205,7 @@
 	if(!ishuman(user))
 		return
 	var/mob/living/carbon/human/human_user = user
-	if(human_user.dna.species.id == SPECIES_HUMAN && !HAS_MIND_TRAIT(human_user, TRAIT_MIMING))
+	if(ishuman(human_user) && !HAS_MIND_TRAIT(human_user, TRAIT_MIMING)) // SKYRAT EDIT CHANGE - ORIGINAL: if(human_user.dna.species.id == SPECIES_HUMAN && !HAS_MIND_TRAIT(human_user, TRAIT_MIMING))
```
Comment (vinylspiders) [inline code/modules/mob/living/emote.dm]:
```suggestion
	if(ishuman(human_user) && !HAS_MIND_TRAIT(human_user, TRAIT_MIMING)) // SKYRAT EDIT CHANGE - Let other species gasp - ORIGINAL: if(human_user.dna.species.id == SPECIES_HUMAN && !HAS_MIND_TRAIT(human_user, TRAIT_MIMING))
```

**Pull request merged**